contracts/src/v0.8/ccip/pools/GHO/UpgradeableBurnMintTokenPool.sol

```diff
@@ -84,6 +84,18 @@ contract UpgradeableBurnMintTokenPool is Initializable, UpgradeableBurnMintToken
     _setRateLimitConfig(remoteChainSelector, outboundConfig, inboundConfig);
   }
 
+  /// @notice This function allows the owner to burn `amount` of the pool's token. This is
+  /// expected to be called while migrating facilitators by offboarding this facilitator in
+  /// favor of a new token pool.
+  /// @dev New token pool should mint and transfer liquidity to this pool (since this pool
+  /// does not hold tokens at any point in time) which can be burnt and hence will reduce
+  /// the facilitator bucket level on GHO. The naming convention mimics that in LockRelease
+  /// type token pools for the sake of consistency.
+  /// @param amount The amount of tokens to burn.
+  function withdrawLiquidity(uint256 amount) external onlyOwner {
```

Comment:
I d prefer to use separate terminology so it's not mixed up with CCIP functionality.
I wonder if it's worth creating a separate contract for this, given that this (ideally) must be implemented by any remote facilitator

Reply:
changed to `burnLiquidity` [0c15ea8](https://github.com/aave/ccip/pull/21/commits/0c15ea8abfff732f921329b19ebe8f2c81d0b692)